File: bakalarkaBE/bakalarkaBE/Models/Ochoreniazdielanie.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace bakalarkaBE.Models;

public partial class Ochoreniazdielanie
{
    public string Zdielajuci { get; set; } = null!;

    public string Cielovy { get; set; } = null!;

    public string Rodnecislo { get; set; } = null!;

    public string Kodochorenia { get; set; } = null!;

    public DateOnly Datumod { get; set; }

    public DateOnly DatumdoZdielanie { get; set; }

    public virtual Doktor CielovyNavigation { get; set; } = null!;
    
    public virtual Pacientoveochorenium Pacientoveochorenium { get; set; } = null!;

    public virtual Doktor ZdielajuciNavigation { get; set; } = null!;
}

File: FE/src/app/models/OchoreniaZdielanie.ts
import {Doktor} from "./Doktor";
import {PacientOchorenie} from "./PacientOchorenie";

export class OchoreniaZdielanie {
  zdielajuci:string = "";
  cielovy:string = "";
  rodnecislo:string = "";
  kodochorenia:string = "";
  datumod:string = "";
  datumdoZdielanie:string = "";
  cielovyNavigation?:Doktor;
  zdielajuciNavigation?:Doktor;
  pacientoveochorenium:PacientOchorenie = new PacientOchorenie();
}
